src/types.md

```diff
@@ -654,6 +654,34 @@ fn to_vec<A: Clone>(xs: &[A]) -> Vec<A> {
 Here, `first` has type `A`, referring to `to_vec`'s `A` type parameter; and
 `rest` has type `Vec<A>`, a vector with element type `A`.
 
+## Anonymous type parameters
+
+> Note: This section is a placeholder for more comprehensive reference
+> material.
+
+> Note: This is often called "impl Trait in argument position".
+
+Functions can declare an argument to be an anonymous type parameter where the
+callee must provide a type that has the bounds declared by the anonymous type
+parameter and the function can only use the methods available by the trait
+bounds of the anonymous type parameter.
```

Comment:
Long sentence without punctuation.

Reply:
I'm not too worried, since it's just there to be a placeholder.